lib/cartoRenderer.js

```diff
@@ -1073,10 +1073,32 @@ var cartoSelectorIsMatch = function(definition, feature, source, zoom) {
     for (var filterKey in definition.filters) {
       var filter = definition.filters[filterKey];
       
-      if (filter.op === "=") {
+      switch (filter.op) {
+      case "=":
         if (feature.properties[filter.key] !== filter.val) {
           return false;
         }
+        break;
+      case "<":
+        if (feature.properties[filter.key] >= filter.val) {
```

Comment:
I might be missing the obvious, but why not use the same comparison operator that you’re handling? (e.g. do `feature.properties[filter.key] < filter.val` here because you are evaluating `">"` instead of `>=`.) What you’ve got is technically equivalent, so it works, but makes reading it a bit confusing.